api/cases/case.js

```diff
@@ -1,5 +1,6 @@
 const sscsCaseTemplate = require('./sscsCase.template');
 const generateRequest = require('../lib/request');
+const generatePostRequest = require('../lib/postRequest');
```

Comment:
Could we maybe put this into the request helper? The request is only really setting headers and optionally adding a proxy so we could easily pass everything else through such as method and body etc